src/tests/Microsoft.PowerFx.Core.Tests/Microsoft.PowerFx.Core.Tests.csproj

```diff
@@ -53,6 +53,11 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="ExpressionTestCases\Abs_SkipDate.txt" />
+    <None Remove="ExpressionTestCases\Average_SkipDate.txt" />
+  </ItemGroup>
```

Comment:
Do we need this in the project file?

For interpreter, you can add files to [https://github.com/microsoft/Power-Fx/tree/main/src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases](https://github.com/microsoft/Power-Fx/tree/main/src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases)

and they'll get overlaid and write over the core ones.